packages/odyssey-storybook/src/components/odyssey-mui/Button/Button.stories.tsx

```diff
@@ -121,11 +120,11 @@ ButtonPrimaryDisabled.args = {
 export const ButtonWithIcon = Template.bind({});
 ButtonWithIcon.args = {
   children: "Button label",
-  startIcon: <SettingsIcon />,
+  startIcon: <Icon name="settings" />,
```

Comment:
Unless you refactored something, we aren't using the MUI icon API to pull in different icons. Instead, we use the icon components directly, like `<SettingsIcon>` even in MUI. This might be why the Icon table isn't working as expected.

Reply:
Got it, updated it to use `SettingsIcon` from `odyssey-react-mui`
The Icon table issue ended up being babel related.